### PR TITLE
Add session expiry section in cfg

### PR DESCRIPTION
**Pull request by drowsy810301**

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -63,6 +63,12 @@
         password=getpass.getpass("Email host's password: ")
     )
 
+if not config.has_section('session_expiry'):
```
Comment (henryyang42) [inline install.py]: I prefer to put it in default config.

**Comment by nickelcheng** [comment]
+1 except for henry's comment.

File: users/views.py [hunk 2]
```diff
@@ -133,9 +133,10 @@ def user_login(request):
             user.backend = 'django.contrib.auth.backends.ModelBackend'
             ip = get_ip(request)
             logger.info('user %s @ %s logged in' % (str(user), ip))
-            one_hour = 60 * 60
-            request.session.set_expiry(one_hour)
-            logger.info('user %s set session timeout one hour' % str(user))
+            hours = int(get_config('session_expiry', 'expiry'))
```
Comment (henryyang42) [inline users/views.py]: Can you store config somewhere else without querying every time?

File: install.py (hunk 1 above)
Comment (bruce3557) [inline install.py]: agree

File: users/views.py (hunk 2 above)
Comment (bruce3557) [inline users/views.py]: We need a singleton instance to get configuration in one instance.

Comment (bruce3557) [inline users/views.py]: If you are not familiar with singleton pattern, please read this and discuss with me or @nickelcheng 
http://stackoverflow.com/questions/6760685/creating-a-singleton-in-python

Comment (bruce3557) [inline users/views.py]: Any problem with this design pattern?

**Comment by bruce3557** [comment]
Going to merge

**Pull request merged**